packages/react/src/components/tooltip/Tooltip.tsx

```diff
@@ -25,14 +25,41 @@ export interface IOverlayTriggerProps {
 }
 
 export interface ITooltipProps extends IOverlayTriggerProps, React.ClassAttributes<Tooltip> {
+    /**
+     * Title of the tooltip

```

Comment:
I find "title" is not really clear enough

```suggestion
     * The text displayed inside the tooltip
```